doc/conf.py

```diff
@@ -65,8 +65,8 @@
 
 # General information about the project.
 project = "atlite"
-author = "The Atlite Authors"
-copyright = "2016-2023" + ", " + author
+author = "Contributors to Atlite"
+copyright = "2016 - 2024" + ", " + author
```

Comment:
```suggestion
copyright = "2016" + ", " + author
```

If you want to include a year, then 2016 is, it seems, the only reasonable choice according to REUSE. Or we get rid of it compeltely.

Reply:
Let's remove it